compiler/foder/README.md

```diff
@@ -0,0 +1,13 @@
+# foder
+
+_foder_ is a header only library that loads files.
+
+## Example
+
+```cpp
+foder::FileLoader fileloader{input_path};
+
+std::vector<char> data = fileloader.load();
+
+const circle::Model *circle_model = circle::GetModel(data.data());
```

Comment:
(Optional) This may confuse users. A _foder_ user is not specific to _circle_ developers ;-)